src/commands/text.ts

```diff
@@ -118,9 +120,14 @@ function getNumberOfSelectedLines(): void {
     if (editor) {
         const numberOfSelectedLines: number = fetchNumberOfSelectedLines(editor);
 
-        (numberOfSelectedLines !== 1)
-            ? window.showInformationMessage(`${numberOfSelectedLines.toString()} Lines Selected`)
-            : window.showInformationMessage(`${numberOfSelectedLines.toString()} Line Selected`);
+        const message = (numberOfSelectedLines !== 1)
+            ? `${numberOfSelectedLines.toString()} Lines Selected`
+            : `${numberOfSelectedLines.toString()} Line Selected`;
+
+        // Show the message to the user
+        window.showInformationMessage(message);
+
+        say.speak(message);
```

Comment:
you could write a wrapper function to make this easier. Like
```
function functionName(message: string) {
    window.showInformationMessage(message);
    say.speak(message);
}
```

Reply:
> you could write a wrapper function to make this easier. Like
> 
> ```
> function functionName(message: string) {
>     window.showInformationMessage(message);
>     say.speak(message);
> }
> ```

Sounds good, will implement in future update.